gradle/projects.main.properties

```diff
@@ -22,3 +22,4 @@ polaris-core=polaris-core
 polaris-service=polaris-service
 polaris-eclipselink=extension/persistence/eclipselink
 aggregated-license-report=aggregated-license-report
+polaris-docker=docker
```

Comment:
Hm - can you move `docker/` to somewhere else or use a nested folder structure?
I could imagine that there'll be more docker related things.